docs/supported-version.md

```diff
@@ -0,0 +1,15 @@
+# Xen Orchestra compatibility
+
+Xen Orchestra is designed to work exclusively on XCP-ng and Citrix XenServer hypervisor. Theorically, Xen Orchestra is fully functionnal with any version of these two hypervisor. However, to benefit from the best support quality, our product is tested to work the best with the following versions:
+
+## Citrix XenServer (Citrix hypervisor)
+
```

Comment:
XenServer 7.5 too

Reply:
Considering XenServer 7.5 has the `VDI_IO_error` should we not wait for the fix?

Reply:
We can't tell we don't support XenServer 7.5 at all :fearful: There is an issue we should link in the doc here, that's Citrix responsibility to deploy a fix for it.